From 93d04538cc6a61b0deb4c85b4bdc66a31fe6e0e3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Chigot?= <clement.chigot@atos.net>
Date: Wed, 21 Apr 2021 16:09:37 +0200
Subject: [PATCH] configure: match AIX major version

Skip minor version of AIX to make RPM common to all TLs for
a AIX major
DO NOT SUBMIT
---
 config.guess | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/config.guess b/config.guess
index 1972fda8e..a019ac3c9 100755
--- a/config.guess
+++ b/config.guess
@@ -630,9 +630,8 @@ EOF
 	else
 		IBM_ARCH=powerpc
 	fi
-	if test -x /usr/bin/lslpp ; then
-		IBM_REV=$(/usr/bin/lslpp -Lqc bos.rte.libc |
-			   awk -F: '{ print $3 }' | sed s/[0-9]*$/0/)
+	if test -x /usr/bin/oslevel ; then
+		IBM_REV=`/usr/bin/oslevel`
 	else
 		IBM_REV="$UNAME_VERSION.$UNAME_RELEASE"
 	fi
-- 
2.25.0

